expression/expr_to_pb.go

```diff
@@ -165,6 +167,14 @@ func ToPBFieldType(ft *types.FieldType) *tipb.FieldType {
 	}
 }
 
+// ToPBFieldTypeWithCheck converts *types.FieldType to *tipb.FieldType with checking the valid decimal for TiFlash
+func ToPBFieldTypeWithCheck(ft *types.FieldType, storeType kv.StoreType) (*tipb.FieldType, error) {
```

Comment:
ToPBFieldTypeWithCheckForTiFlash?

Reply:
storeType can also be tikv, if updated name maybe confuse us.